src/components/pager/pager.component.js

```diff
@@ -57,6 +56,7 @@ const Pager = (props) => {
         { ...props }
         currentPage={ currentPage }
         setCurrentPage={ setCurrentPage }
+        currentTheme={ setCurrentTheme }
```

Comment:
Shouldn't this use the getter?

Reply:
No it's a callback to allow the navigation sub component to set the theme in the parent, it probably needs a better name for the prop though

Reply:
What do you think if I rename the prop to `setcurrentThemeName` and pass in the hook setter as a callback to it?

Reply:
Sorry, yep gotcha. Yeah that reads a bit more clearly 👍